src/utils/applyTheme.ts

```diff
@@ -6,6 +6,8 @@ export type Document = {
   [key: string]: any;
 };
 
+const THEME_STYLE_TAG_ID = "visual-editor-theme" as const;
```

Comment:
`as const` is required for a const?

Reply:
The `as const` sets the type of THEME_STYLE_TAG_ID as `"visual-editor-theme"` vs. `string`
Shouldn't really matter so I can drop it if it just adds clutter

Reply:
I don't think the type matters since it's just used as a string so we can drop it.